packages/svelte/messages/compile-errors/script.md

```diff
@@ -0,0 +1,135 @@
+## bindable_invalid_location
+
+> `$bindable()` can only be used inside a `$props()` declaration
+
+## constant_assignment
+
+> Cannot assign to %thing%
+
+## constant_binding
+
+> Cannot bind to %thing%
+
+## declaration_duplicate
+
+> `%name%` has already been declared
+
+## declaration_duplicate_module_import
+
+> Cannot declare same variable name which is imported inside `<script context="module">`
+
+## derived_invalid_export
+
+> Cannot export derived state from a module. To expose the current derived value, export a function returning its value
+
+## dollar_binding_invalid
+
+> The $ name is reserved, and cannot be used for variables and imports
+
+## dollar_prefix_invalid
+
+> The $ prefix is reserved, and cannot be used for variables and imports
+
+## each_item_invalid_assignment
+
+> Cannot reassign or bind to each block argument in runes mode. Use the array and index variables instead (e.g. `array[i] = value` instead of `entry = value`)
+
+## effect_invalid_placement
+
+> `$effect()` can only be used as an expression statement
+
+## global_reference_invalid
+
+> `%name%` is an illegal variable name. To reference a global variable called `%name%`, use `globalThis.%name%`
+
+## host_invalid_placement
+
+> `$host()` can only be used inside custom element component instances
+
+## legacy_export_invalid
+
+> Cannot use `export let` in runes mode — use $props instead
```

Comment:
(in general runes are sometimes written with and sometimes without the braces, I'm not sure if it makes sense to unify it or if it makes more sense to leave it in/out on a case by case basis. Even if you disagree with the braces change, the backticks change should go in)
```suggestion
> Cannot use `export let` in runes mode — use `$props()` instead
```